quickstarts/Models.ipynb

```diff
@@ -11,7 +11,7 @@
     },
```

Comment:
<p>Please remove drop the comment line, as the preceding text should do the explaining.</p><p> </p><p>And instead of calling the variable <code>model</code>, can you call it <code>model_info</code> or something similar? We use <code>model</code> when we document <code>genai.GenerativeModel()</code>, so I want to make sure it's clear that this is a different class and that <code>model.generate_content</code> won't work here. </p><p> </p><p>Also:</p><ul><li>Add spaces around the <code>=</code></li><li>Remove the trailing <code>;</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Thanks Mark ! 
Removed comment and updated variable name and removed trailing ; and gave space between =